### DISCUSSION TITLE
Mail

### DISCUSSION THREAD

**Discussion by ghost** [community/community#28216] [Other Feature Feedback, Questions, & Ideas]
I cannot verify a new email on GitHub

**Comment by atuschek0** [comment]
Do you have another email address you can use? Try using a different browser possibly.

**Comment by ghost** [comment]
i have but unfortunately i cant verify in and also not visible the email
verification

**Comment by jfullstackdev** [comment]
if you are just adding it for the sake of linked commits, you can leave it like that, all the associated commits will be added to your account ( and badges too ! )
